.env.ci

```
@@ -1,6 +1,6 @@
 MODE=development
 ORIGIN=http://localhost/
-DATABASE_URL=postgresql://localhost/hackerspub
+DATABASE_URL=postgresql://postgres:password@localhost:5432/hackerspub
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the MODE key

(UnorderedKey)